src/process/sys/linux/process.rs

```diff
@@ -133,10 +133,10 @@ impl Process {
 				// TODO: fix or document the unwraps
 				let fd = path
 					.file_name()
-					.unwrap()
+					.ok_or(ProcessError::ParserError { pid: self.pid })?
```

Comment:
After doing some research, I think we can leave this as an unwrap since directory entries should always contain a file name. Or actually we should change it to an `expect` with a message.